applications/GeoMechanicsApplication/tests/cpp_tests/test_compressibility_matrix.cpp

```diff
@@ -0,0 +1,87 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#include "containers/model.h"
+#include "custom_utilities/transport_equation_utilities.hpp"
+#include "includes/checks.h"
+#include "testing/testing.h"
+#include <boost/numeric/ublas/assignment.hpp>
+
+using namespace Kratos;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(CalculateCompressibilityMatrix2D3NGivesCorrectResults, KratosGeoMechanicsFastSuite)
+{
+    // clang-format off
+    Vector Np(3);
+    Np <<= 1.0, 2.0, 3.0;
+    // clang-format on
+
+    BoundedMatrix<double, 3, 3> CompressibilityMatrix  = ZeroMatrix(3, 3);
```

Comment:
Unfortunately, the style guide says local variables should be snake case:
```suggestion
    BoundedMatrix<double, 3, 3> compressibility_matrix = ZeroMatrix(3, 3);
```
There are a few more of these name convention differences in this test file.

Reply:
Ops, changed all local variables into a snake format.